programmers/6일차/체육복.js

```diff
@@ -0,0 +1,24 @@
+function solution(n, lost, reserve){
+    const trueReserve = reserve.filter( x => !lost.includes(x));
+    const trueLost = lost.filter( y => !reserve.includes(y));
+    
+    trueLost.sort((a,b) => a-b);
+    trueReserve.sort((a,b) => a-b);
+    console.log(trueLost);
+
+    for (let i of trueReserve) {    
```

Comment:
```suggestion
    for (let student of trueReserve) {    
```
i 보다는 이게 더 좋을 것 같습니다  for of 나 in 에서는 i j k 를 잘 쓰지 않아요